test/e2e/util/utils.go

```diff
@@ -391,6 +416,190 @@ func DeployEraserHelm(namespace string, args ...string) env.Func {
 	}
 }
 
+func GetManagerLogs(ctx context.Context, cfg *envconf.Config, t *testing.T) error {
+	c, err := cfg.NewClient()
+	if err != nil {
+		return err
+	}
+
+	var pods corev1.PodList
+	err = c.Resources().List(ctx, &pods, func(o *metav1.ListOptions) {
+		o.LabelSelector = labels.SelectorFromSet(map[string]string{"control-plane": "controller-manager"}).String()
+	})
+	if err != nil {
+		return err
+	}
+
+	if len(pods.Items) > 1 {
+		return errors.New("only one manager pod should be present")
+	} else if len(pods.Items) == 0 {
+		return errors.New("no manager pod present")
+	}
+
+	manager := pods.Items[0]
+
+	output, err := KubectlLogs(cfg.KubeconfigFile(), manager.Name, "", EraserNamespace)
+	if err != nil {
+		return err
+	}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		return err
+	}
+
+	testName := strings.Split(t.Name(), "/")[0]
+
+	// get log output file path
+	path := filepath.Join(wd, TestLogDir, testName)
+
+	var file *os.File
+	if !fileExists(filepath.Join(path, manager.Name)) {
+		if err := os.MkdirAll(path, filemode); err != nil {
```

Comment:
This needs 0700 at least, probably 0755 would work best.
You can also just call `os.MkdirAll` without needing to check `fileExists`